src/signature.c

```diff
@@ -137,7 +135,10 @@ PyDoc_STRVAR(Signature_name__doc__, "Name.");
 PyObject *
 Signature_name__get__(Signature *self)
 {
-    return to_unicode(self->signature->name, self->encoding, NULL);
+    return to_unicode(
+        self->signature->name,
+        value_or_default(self->encoding, "utf-8"),
+        NULL);
```

Comment:
This change is not needed, since `to_unicode` already handles the case when the given `encoding` is `NULL`.